Diagnostics/AVTester/Write-SimpleLogFile.ps1

```diff
@@ -51,8 +54,15 @@ function Write-SimpleLogFile {
         [Parameter(Mandatory = $true)]
         [string]$String,
 
+        [parameter(Mandatory)]
+        [ValidateScript({
+                if (Test-Path -Path $_ -PathType Container) { $true }
+                else { throw "Path $_ is not valid" }
+            })]
+        [String]$Path,
+
```

Comment:
This is going to break `Set-ExchAVExclusions.ps1` the way you have this setup. By creating a new mandatory parameter to `Write-SimpleLogFile` that is used in this other script, we need to also modify that script or update it so it accepts a default value.

Reply:
The syntax of `Some-Command -Path C:\must\be\a\folder -FileName MustBeAFile.log` is very unusual. Unless we have a justification for this, I suggest changing to the same syntax as every other command that takes a file, which is to use a single parameter:

```
PS C:\> Start-Transcript -Path C:\Users\bilong\Documents\file.log
Transcript started, output file is C:\Users\bilong\Documents\file.log
PS C:\> Get-Process | Export-Csv -Path C:\Users\bilong\Documents\export.csv
PS C:\> Add-Content -Path C:\Users\bilong\Documents\export.csv -Value "Foo"
```

Reply:
> This is going to break `Set-ExchAVExclusions.ps1` the way you have this setup. By creating a new mandatory parameter to `Write-SimpleLogFile` that is used in this other script, we need to also modify that script or update it so it accepts a default value.

I miss that part